packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -228,7 +228,7 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
 
         await assert.rejects(
             container.attach(provider.driver.createCreateNewRequest(provider.documentId)),
-            /(attaching container with blobs is not yet implemented)|(create empty file not supported)/,
```

Comment:
this looks wrong. not sure what happened here

Reply:
So the test matches the error thrown as either one of those 2 strings. Since now those are replaced with codes, we have to replace it with codes here.

Reply:
I have to manually add this change.

Reply:
ohhhhh. i see. i thought this was auto updated. looks good then